<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SipAccountOptionsWidget</class>
 <widget class="QWidget" name="SipAccountOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>248</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="grbSipAccount">
     <property name="title">
      <string>Account</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lblSipLogin">
        <property name="text">
         <string>Login:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lneSipLogin"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblSipPassword">
        <property name="text">
         <string>Password:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lneSipPassword">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grbSipConnection">
     <property name="title">
      <string>Connection</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="lblSipServerHost">
        <property name="text">
         <string>SIP Server:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lneSipServerHost"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="lblSipServerPort">
        <property name="text">
         <string>Port:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QSpinBox" name="spbSipServerPort">
        <property name="maximum">
         <number>65535</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblSipProxyHost">
        <property name="text">
         <string>SIP Proxy:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lneSipProxyHost"/>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="lblSipProxyPort">
        <property name="text">
         <string>Port:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QSpinBox" name="spbSipProxyPort">
        <property name="maximum">
         <number>65535</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grbSipMedia">
     <property name="title">
      <string>Media Streams</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="lblSipStun">
          <property name="text">
           <string>STUN Server:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lneSipStun"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="chbSipIce">
        <property name="text">
         <string>Enable ICE negotiation</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
